seed/challenges/01-front-end-development-certification/intermediate-bonfires.json

```diff
@@ -663,7 +663,7 @@
       "title": "Sum All Primes",
       "description": [
         "Sum all the prime numbers up to and including the provided number.",
-        "A prime number is defined as having only two divisors, 1 and itself. For example, 2 is a prime number because it's only divisible by 1 and 2. 1 isn't a prime number, because it's only divisible by itself.",
+        "A prime number is defined as a number greater than one and having only two divisors, one and itself. For example, 2 is a prime number because it's only divisible by one and two. 1 isn't a prime number, because it's only divisible by itself.",
```

Comment:
"A prime number is defined as a number greater than one" and "1 isn't a prime number, because it's only divisible by itself." Contradict each other. I think the first line is sufficient enough so we can remove the last line.